### PR TITLE
Dataset standard encryption

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**
update documentation with dataset standard encryption changes (target release v6)

### CODE REVIEW

File: for-developers/confidential-computing/sgx-encrypted-dataset.md [hunk 1]
```diff
@@ -53,59 +53,51 @@ This command will create the folders `datasets/encrypted`, `datasets/original` a
 ...
 ```
 
-First, create your dataset folder:
+We will create a dummy file that has `"Hello, world!"` as content inside `datasets/original`. Alternatively, you can put your own dataset file.
```
Comment (james-toussaint) [inline for-developers/confidential-computing/sgx-encrypted-dataset.md:56]:
```suggestion
We will create a dummy confidential file that has `"My Social Security Number is 721-07-4426. Don't tell anyone."` as content inside `datasets/original`. Alternatively, you can put your own dataset file.
```

File: for-developers/confidential-computing/sgx-encrypted-dataset.md [hunk 2]
```diff
@@ -171,11 +163,12 @@ const figlet = require('figlet');
   try {
     const iexecOut = process.env.IEXEC_OUT;
     const iexecIn = process.env.IEXEC_IN;
+    const datasetFileName = process.env.IEXEC_DATASET_FILE_NAME;
```
Comment (james-toussaint) [inline for-developers/confidential-computing/sgx-encrypted-dataset.md:166]: :+1:

File: for-developers/your-first-app.md [hunk 3]
```diff
@@ -105,15 +110,19 @@ result.zip
...
-* If your app produces output files, you must copy them in `/iexec_out/` .
+* `$IEXEC_OUT/computed.json` file should be created when your computing is over. It contains at least a field `deterministic-output-path` which is the path of the deterministic portion of your results. It should be either a file or a non-empty folder. It is required for the proof of execution \(given the same inputs this file should always be the same\).
+* If your app produces output files, you must copy them in the iExec output directory.
 
 {% hint style="warning" %}
-Your application must always create a `computed.json` file in `/iexec_out` as a proof of execution which could look like `{ "deterministic-output-path" : "/iexec_out/result.txt" }`
+Your application must always create a `computed.json` file in the iExec output directory as a proof of execution which could look like `{ "deterministic-output-path" : "/iexec_out/result.txt" }`
```
Comment (james-toussaint) [inline for-developers/your-first-app.md:123]: Probably should mention callback as a possible option 2 (in this case deterministic-output-path" isn't required)

**Review by james-toussaint — APPROVED** [review]

**Review by zguesmi — APPROVED** [review]
:+1:

**Pull request merged**